templates/terraform/decoders/global_neg_fetch_endpoint.go.erb

```diff
@@ -0,0 +1,36 @@
+config := meta.(*Config)
```

Comment:
This isn't covered by the normal NestedQuery decoder? From my quick lookover, it looks similar but just has less extra comparison steps. If you ran into trouble, feel free to keep this - I'm just curious

Reply:
This was actually a mistake, I should have deleted this file. It was from when I was looking at implementing group + endpoint as a single resource that only allowed a single endpoint per group.

With separate endpoints + groups this is unused